apps/ledger-live-mobile/src/reducers/accounts.ts

```diff
@@ -450,6 +450,23 @@ export const orderedNftsSelector = createSelector(
   (accounts, nfts) => orderByLastReceived(accounts, nfts),
 );
 
+export const hiddenNftsSelector = createSelector(
+  hiddenNftCollectionsSelector,
+  accountsSelector,
+  (hiddenNFts, accounts) =>
+    hiddenNFts.reduce<{ nft: ProtoNFT; id: string }[]>((p, c) => {
```

Comment:
This logic has been removed from the components and put here. However it is worth noting it behaves slightly differently. Previously the hidden NFT list would render hidden NFT's that were no longer in your account but would fail to show the name/image because the metadata request could not be made. Now instead, we only show hidden NFT's in this list if it actually exists in your account still.